samples/Basic YARP Sample/MyStartup.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace BasicYARPSample
+{
+    // Provides a configuration for the kestrel server with reverse proxy middleware
+    public class MyStartup
```

Comment:
Startup is a very standard convention in AspNet. Changing it is likely to cause confusion when comparing to other samples.
```suggestion
    public class Startup
```

Reply:
IMHO both are a really bad name, as its not really startup, its more of a configuration. ServiceInit or something would be better.

Reply:
Granted, but let's stick with the standards where can.